### DISCUSSION TITLE
One click for move skin interface osu! (triangles) to any other skin.

### DISCUSSION THREAD

**Discussion by NeFender** [ppy/osu#20074] [Ideas] [answered]
I mean health, combo, pp counter, progress bar of beatmap. But hitcircles will remain unchanged. Skin interface osu! (triangles) I really liked it, but manually transferring it to another skin will take a lot of time.
Google Translate

**Comment by peppy** [comment, marked as answer]
We definitely want to make copying elements between skins a whole lot easier, so you can look forward to that.
